src/Send/SendScreen/SendScreen.tsx

```diff
@@ -157,25 +159,18 @@ export const SendScreen = ({
   const handleConfirm = async () => {
     if (!isValid || recomputing || !yoroiUnsignedTx) return
 
-    const defaultAssetAmount = tokenInfo.isDefault
-      ? parseAmountDecimal(amount, tokenInfo)
+    const defaultAssetAmount: Quantity = tokenInfo.isDefault
+      ? (parseAmountDecimal(amount, tokenInfo).toString() as Quantity)
       : // note: inside this if balanceAfter shouldn't be null
-        tokenBalance.getDefault().minus(balanceAfter ?? 0)
+        Quantities.diff(defaultAssetAvailableAmount, balanceAfter ?? '0')
```

Comment:
if this is the amount of ADA being send, get that directly from `yoroiUnsignedTx`

Reply:
That is the amount of ADA spent when what is going to be sent is a token.

`balanceAfter` = Total ADA on balance  - fee - minADA 

ADA to send => Total ADA on balance - balanceAfter = fee + minADA